react/base/src/index.tsx

```diff
@@ -1,10 +1,22 @@
+import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
+import * as serviceWorkerRegistration from './serviceWorkerRegistration';
+import reportWebVitals from './reportWebVitals';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+// Learn more about service workers: https://cra.link/PWA
+serviceWorkerRegistration.unregister();
```

Comment:
Should we make this `register` by default?

Reply:
probably not. Its this way in CRA, and turning it on would mean you get SW while DEVing, which isn't fun

Reply:
In the register, there's a check for prod builds

``
if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
```